tasklib/task.py

```diff
@@ -333,7 +333,8 @@ def serialize_depends(self, cur_dependencies):
 
     def delete(self):
         if not self.saved:
-            raise Task.NotSaved("Task needs to be saved before it can be deleted")
+            raise Task.NotSaved(
+                "Task needs to be saved before it can be deleted")
```

Comment:
2 things to point out here for ultimate code compliance 😄 

1. Strings should be single-quoted where possible (that is, unless the string itself contains a single quote).
2. If you start a new line after an opening parenthesis `(`, close the parentheses on their own line too (and remember the trailing commas!).

So:

```python
raise Task.NotSaved(
    'Task needs to be saved before it can be deleted',
)
```

Note that if the string is too long and causes the line to be longer than 80 characters, use implicit string concatenation:

```python
raise Task.NotSaved(
    'Task needs to be saved before it can be deleted, because '
    ' of some reason that makes the string really long!',
)

Reply:
Fixed in 8f4fabf

Reply:
Randomly stumbling here, I wonder @robgolding why the preference for single quotes?

Reply:
Simply a personal preference @tbabej, which is not documented in any coding guidelines (yet!) 😄